src/Sylius/Bundle/WebBundle/Resources/public/js/sylius-attribute-option-list.js

```diff
@@ -0,0 +1,71 @@
+(function ($, window, document) {
+    var $collectionHolder;
+
+    jQuery(document).ready(function() {
```

Comment:
You're scoping the global `jQuery` as `$`, yet you're using the global here.

Change it to `$(document).ready(function () {`

That said, this might be copy-and-paste - in which case, we might look at preventing duplication later on.